gobblin-temporal/src/main/java/org/apache/gobblin/temporal/ddm/launcher/ExecuteGobblinJobLauncher.java

```diff
@@ -83,10 +83,12 @@ public ExecuteGobblinJobLauncher(
   public void submitJob(List<WorkUnit> workunits) {
     try {
       Properties finalProps = adjustJobProperties(this.jobProps);
+      // Initialize workflowId to be used by cancel workflow.
```

Comment:
Tiny nit: technically speaking the workflow ID isn't just used by cancel workflow right, it's just used to track any top-level temporal workflow kicked off by this launcher.